app/views/pages/faq.html.haml

```diff
@@ -26,6 +46,12 @@
     The offline caching system is pretty rudimentary at the moment. It downloads the original webpage, and attempts to grab all the directly referenced script, css and image tags within that webpage. In the off chance that your bookmarked webpage is to a site that actively tries to prevent crawlers, or a site that is rendered with flash or even JS (looking at you, react rendered blog), transientBug is unable to cache those sites at this time.
 
 %p
-  %b.question Why am I getting a 500 error for some caches?
+  %b.question Why am I getting an error for some caches?
+  .answer
+    Tl;dr: No one is perfect. Chances are the page redirects too many times for transientBug to follow, or isn't a valid URL. I'm working on making transientBug smarter about handling these but sometimes that isn't good enough and the only thing transientBug can do is throw an error.
```

Comment:
> Tl;dr: No one is perfect. Chances are the page redirects too many times for transientBug to follow, or isn't a valid URL. I'm working on making transientBug smarter about handling these but sometimes that isn't good enough and the only thing transientBug can do is throw an error. 

~Tl;dr~ because there is no other, longer explanation.
~No one is perfect.~ how about`Caching is hard.`? 
~I'm working on making transientBug smarter about handling these~ how about `transientBug is getting smarter and better` I think it's more readable if you talk about TB in the third person ("transientBug is") rather than yourself in the first person and TB as an object ("I'm working on transientBug"). It's also more consistent with the second half of the sentence: `the only thing transientBug can do is`
~throw an error~ how about `complain loudly`